src/api/app/models/package.rb

```diff
@@ -1457,6 +1460,28 @@ def last_build_reason(repo, arch, package_name = nil)
     PackageBuildReason.new(data)
   end
 
+  # Methods for Thinking Sphinx real-time indexes
+
+  def attribs_attrib_type_ids
+    attribs.pluck(:attrib_type_id)
+  end
+
+  def package_issues_ids
+    package_issues.pluck(:issue_id)
+  end
+
+  def devel_packages?
+    develpackages.exists?
+  end
+
+  def backend_packages?
+    Package.find_by_sql("SELECT * FROM backend_packages WHERE package_id = #{id}").present?
```

Comment:
This is a [cache](https://github.com/openSUSE/open-build-service/wiki/The-BackendPackage-Cache), you know this right?

Reply:
I changed this line to `backend_package.present?`, making use of the `has_one` relationship.

Being a cache, do you see it could be harmfull to use this query using real-time indices?

Reply:
Well potentially, if the case doesn't exist during your `backend_package.present?` call, it will be [created](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/package.rb#L1107).